### PR TITLE
Ecology

### PR DESCRIPTION
**Pull request by yvanlebras**
Following comments and help in #998 , here is a firts PR to the master branch for 2 ecology related tutorials.

I : 
- filled the metadata on the top of the tutorial.md and *.html files, even if the slides.html file is not ready for use for now, just a template....
- movd the images inside topics/ecology/tutorials/species-distribution-modeling/Images as inside topics/ecology/tutorials/regionalGAM/Images to topics/ecology/images?
- choosed to don't move the the tutorials not named tutorial.md to their one tutorial for now but have to think more about it... Maybe the best will be to refusion both in one, but this seems to me hard to follow... So maybe not considering the multispecies approach is better in a first glance
- made the PR against master branch; this PR ;)

### CODE REVIEW

**Comment by yvanlebras** [comment]
@bebatut can you have a look? As you see, we are really not on a "last minute" PR, just, "last day" ;)

File: topics/ecology/tutorials/species-distribution-modeling/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+objectives:
+    - "Find and download occurrences data from GBIF"
+    - "Find and download environmental data"
+    - "Process both occurrences and environmental"
+    - "Partition occurrence data"
+    - "Model a theoretical ecological niche and predict species distribution in a future climate scenario by using SDM"
+requirements:
+time_estimation: "1h"
+key_points:
+    - ""
```
Comment (shiltemann) [inline topics/ecology/tutorials/species-distribution-modeling/tutorial.md]: do you have any conclusions/take-home messages to add here? (and in other tutorial)

File: topics/ecology/tutorials/regionalGAM/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+This shows the abundance of *Pyronia tithonus*, over the weeks. We can see there is a peak every year as expected and we can notice some differences in the values between the weeks and years, but not how the weeks overlap between the years. We will plot a stalked version of this visualization to check if there are no unexpected changes and have an idea of the quality of the reconstructed phenology.
+
+> ### {% icon hands_on %} Hands-on: Visualize the years on a stacked graph
+> 1. **Scatterplot w ggplot2** {% icon tool %} with the following parameters
+>    - {% icon param-file %} *"from dataset"*: output from **Flight curve**
+>    - *"Column to plot on x-axis"*: `4` (the column with the year)
+>    - *"Column to plot on y-axis"*: `6`
+>    - *"Plot title"*: add a meaningful title (e.g. `Pyronia tithonus phenology stacked visualization`)
+>    - *"Label for x axis"*: add a meaningful label (e.g. `Day number`)
+>    - *"Label for y axis"*: add a meaningful label <!-- TODO (e.g. ????? Ask Benjamin or Reto to know more)-->
```
Comment (shiltemann) [inline topics/ecology/tutorials/regionalGAM/tutorial.md]: do you know this axis label yet?

**Comment by shiltemann** [comment]
Thanks @yvanlebras! there were some TODOs left, but I will merge this before your workshop this afternoon so I commented them out for now, maybe you can add them during the hackathon? ;)
